backend/app/Models/Stack.php

```diff
@@ -10,6 +10,6 @@ class Stack extends Model
 {
     use HasFactory, Uuids;
 
-    protected $fillable = ["name"];
+    protected $fillable = ['id', 'name'];
```

Comment:
why are you filling name here? this would be done by the DB.

Reply:
name of stack has to be create by user